sdk/identity/azure-identity/azure/identity/_credentials/azure_cli.py

```diff
@@ -139,14 +138,13 @@ def parse_token(output) -> Optional[AccessToken]:
     """
     try:
         token = json.loads(output)
-        dt = datetime.strptime(token["expiresOn"], "%Y-%m-%d %H:%M:%S.%f")
-        if hasattr(dt, "timestamp"):
-            # Python >= 3.3
-            expires_on = dt.timestamp()
-        else:
-            # taken from Python 3.5's datetime.timestamp()
-            expires_on = time.mktime((dt.year, dt.month, dt.day, dt.hour, dt.minute, dt.second, -1, -1, -1))
 
+        # Use "expires_on" if it's present, otherwise use "expiresOn".
+        if "expires_on" in token:
+            return AccessToken(token["accessToken"], int(token["expires_on"]))
```

Comment:
Are you expecting AccessToken's expires_on units to be milliseconds or seconds? 

The [docs](https://github.com/pvaneck/azure-sdk-for-python/blob/669992060b05579d45334b34b7a923ac85114e46/sdk/core/azure-core/azure/core/credentials.py#L19) don't specify but in JS we expect milliseconds.

If you expect milliseconds, I believe the az cli value is actually in seconds so you might need to convert it?

Reply:
Here, we are expecting seconds, so I believe we are good on not having a unit mismatch. Good callout.😺